text.cabal

```diff
@@ -76,15 +76,13 @@ flag developer
 flag simdutf
   description: use simdutf library, causes Data.Text.Internal.Validate.Simd to be exposed
   default: True
-  manual: True
 
 flag pure-haskell
   description: Don't use text's standard C routines
     When this flag is true, text will use pure Haskell variants of the
     routines. This is not recommended except for use with GHC's JavaScript
     backend.
   default: False
-  manual: True
```

Comment:
This and `simduft` flag **must be** manual.

Reply:
I am curious, is there ever a good time to make a flag automatic?

Reply:
If you are encoding something like

```
if build-depends(aeson >= 2.1)
  build-depends: attoparsec-aeson ^>=2.1
```

i.e.

```
if flag(aeson-2-1)
  build-depends: aeson >= 2.1
  build-depends: attoparsec-aeson ^>=2.1
else
  -- disjoint constraint to force flag selection
  build-depends: aeson < 2.1
```